### DISCUSSION TITLE
How to add an icon in 1.6.2

### DISCUSSION THREAD

**Discussion by RedMik52** [KadeArchive/Kade-Engine#1822] [Q&A] [answered]
Can someone help me with this problem bc i'm trying to add an icon but idk how to name it to make it work

**Comment by napstaa967** [comment, marked as answer]
make a new file called "icon-< character name >"  that's 300x150 then add the icons to it, each has to be 150x150, first one has to be the normal one second one the dead one

if the character has a hyphen in the name:
open healthicon.hx
modify this line and add:
```hx
if (char != 'bf-pixel' && char != 'bf-old' && char != 'character name')
```

**Reply by RedMik52** [reply]
Thanks a lot for the help
